src/scss/constants.scss

```diff
@@ -24,6 +24,7 @@ $input--invalid-focus: #ff7384;
 $input--invalid-box-shadow-focus: 0 0 0 2px rgba($input--invalid-focus, 1);
 
 // @debt replace all greyish colors with $secondary alternatives
+$schedule-opened-grey: #4a4342;
```

Comment:
is this value needed? can't find where it's used.

Reply:
yep, used in 
![image](https://user-images.githubusercontent.com/56254806/130732984-bfbf7d0d-e958-4449-aade-0df3e6a0cf2a.png)